Assets/LeapMotion/Core/Scripts/Switches/Internal/Updater.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using UnityEngine;
+
+namespace Leap.Unity {
+
+  public class Updater : MonoBehaviour {
```

Comment:
This looks like this might be a concept orthogonal to switches, maybe it could be moved into the general utilities instead?